tests/test_06_l2vpn_return_codes.py

```diff
@@ -95,6 +93,7 @@ def test_013_create_l2vpn_with_vlan_untagged_code201(self):
         response = requests.post(api_url, json=payload)
         assert response.status_code == 201, response.text
 
+    @pytest.mark.xfail(reason="return status 400 -> Validation error: Scheduling not possible: 2025-01-21 21:01:08.149547+00:00 start_time cannot be before the current time\nScheduling not possible: 2025-12-31T12:00:00Z end_time is not in a valid ISO format")
```

Comment:
@gretelliz can you please open an issue about this error of start_time not being before the current time? According to the Provisioning spec, the user can request a circuit without specifying the start_time.